source/whats-new.txt

```diff
@@ -39,6 +41,19 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-6.0:
+
+What's New in 6.0
+-----------------
+
+The {+driver-short+} v6.0 release includes the following features:
+
+- This release removes support for the ``addUser()`` helper command. Use the
+  :manual:`createUser</reference/command/createUser>` command instead.
+- This release removes support for the ``collStats`` helper command. Use the
```

Comment:
Question:

I'm not sure what a "helper command" is. Would "operation" be a better description? I see that it's called "operation" in the [related PR](https://github.com/mongodb/node-mongodb-native/pull/3638).

I think it would probably be good to differentiate from addUser() since I think that's a method you can call from a [database](https://mongodb.github.io/node-mongodb-native/5.4/classes/Db.html#addUser) or [Admin](https://mongodb.github.io/node-mongodb-native/5.4/classes/Admin.html#addUser) instance.

I see that it's also mentioned in the admonition for 5.4, so if you change it, it would need to be updated there too.

Reply:
will change admonition in a separate ticket to avoid backporting issues